hardhat.config.ts

```diff
@@ -1,9 +1,11 @@
 /**
  * @type import('hardhat/config').HardhatUserConfig
  */
-require('dotenv').config()
 import '@nomiclabs/hardhat-ethers'
 import '@nomiclabs/hardhat-waffle'
+require("./tasks/deployMerkleDistributor");
+
+const { vars } = require("hardhat/config");
```

Comment:
Use ES modules imports instead. They are widely supported already.
```suggestion
import { vars } from "hardhat/config";
import "./tasks/deployMerkleDistributor";
```

Reply:
```
[# ESM and TypeScript projects](https://hardhat.org/hardhat-runner/docs/advanced/using-esm#esm-and-typescript-projects)
At the moment, it's not possible to use ESM in TypeScript projects.
```
I found this when i originally tried to do it and didnt bother, but is seems to work so I followed your suggestion

Reply:
Yeah, that's weird.
Honestly it doesn't matter, however I saw ES6 Modules syntax there and thought it would be better to standardize it: use either CJS or ES6 Modules.